features/shared/checkout_previous_branch_after_success/new_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that creates a new current branch intact and leaves the previous branch intact
```

Comment:
Same as https://github.com/Originate/git-town/pull/501#discussion_r29312041

So in this case:
> Feature: when a new branch is created during a Git Town command, the current branch becomes the new previous branch